src/backend/api/v1/Users.ts

```diff
@@ -120,4 +122,14 @@ usersRouter.delete('/deleteAvatar', (req, res, next) => {
     fs.unlink('./public' + req.body.oldPath.substring(1, req.body.oldPath.length), (err) => {
         res.send('Deleted')
     });
-});
\ No newline at end of file
+});
+
+//recommend user algorithm start
+usersRouter.get('/recommend/:_id', async (req, res, next) => {
+    const userId = req.params._id;
+    const currentUserSchema = await User.find({_id: userId}).exec();
+    const currentUser = currentUserSchema[0];
+    const userList = await User.find({_id: { $ne: userId}}).exec(); 
```

Comment:
Minor: Maybe we can get the users have at least 1 same tag or the same major as the user. It might help to decrease the # of queried users.

Reply:
Sure, I will edit it later when it is needed